src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: $day-gap;
+  max-width: ($day-size * 7) + ($day-gap * 6);
+
+  @each $day, $count in $days {
+    &--start-day-#{$day} :first-child {
+      margin-left: ($day-size + $day-gap) * $count;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there should be no spaces inside parentheses. Please remove the space after the opening parenthesis and before the closing parenthesis.